lms/static/js/discovery/app.js

```diff
@@ -1,70 +1,72 @@
 ;(function (define) {
-
-define(['backbone', 'course_discovery_meanings'], function(Backbone, meanings) {
     'use strict';
 
-    return function (Collection, Form, ResultListView, FilterBarView, FacetsBarView, searchQuery) {
-        //facet types configuration - set default display names
-        var facetsTypes = meanings;
+    define(['backbone', 'js/discovery/collection', 'js/discovery/form', 'js/discovery/result_list_view',
```

Comment:
Actually, maybe just change it to be "main.js" to be consistent with the other 2 mains in this PR. I was thinking you touched all of them in the cleanup, but instead it is just discovery.